common/static/common/js/discussion/utils.js

```diff
@@ -384,7 +384,7 @@
                 } else if (RE_DISPLAYMATH.test(htmlString)) {
                     htmlString = htmlString.replace(RE_DISPLAYMATH, function($0, $1, $2, $3) {
                         /*
-                         bug fix, ordering is off
+                         correct order in preview for mathjax
```

Comment:
should this comment really be something like, "needed to ensure that mathjax is correctly ordered in preview"?